user-interface/src/main/java/life/qbic/datamanager/views/projects/project/samples/registration/batch/BatchRegistrationDialog.java

```diff
@@ -241,4 +235,15 @@ public BatchRegistrationContent batchRegistrationContent() {
   public List<SampleRegistrationContent> sampleRegistrationContent() {
     return sampleSpreadsheetLayout.getContent();
   }
+
+  public static class CancelEvent extends UserCancelEvent<BatchRegistrationDialog> {
+
+    @Serial
+    private static final long serialVersionUID = 8348902371944310641L;
+
+    public CancelEvent(BatchRegistrationDialog source, boolean fromClient) {
+      super(source, fromClient);
+    }
+  }
```

Comment:
Is this distinction really necessary? Where is the cancelevent employed?

Reply:
I don't know. The whole dialog is removed now.